debian/appscale_install_functions.sh

```diff
@@ -18,7 +18,11 @@ fi
 VERSION_FILE="$APPSCALE_HOME_RUNTIME"/VERSION
 export APPSCALE_VERSION=$(grep AppScale "$VERSION_FILE" | sed 's/AppScale version \(.*\)/\1/')
 
+<<<<<<< HEAD
```

Comment:
There are merge issues in multiple files. Please fix this and run regression testing.